### PR TITLE
add key features to track bash

### LINKED ISSUE #469: Add key features
_This issue is part of the migration to v3. You can read full details about the various changes [here](https://github.com/exercism/v3-launch/)._

In Exercism v3, each track must specify exactly six "key features". Exercism uses these features to highlight the most interesting, unique or "best" features of a language to a student.

Key features are specified in the top-level `"key_features"` field in the track's `config.json` file and are defined as an array of objects, as specified in [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#key-features).

## Goal

The `"key_features"` field in the `config.json` file should be updated to describe the six "key features" of this track. See [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#key-features).

## Example

```json
{
  "key_features": [
    {
      "icon": "features-oop",
      "title": "Modern",
      "content": "C# is a modern, fast-evolving language."
    },
    {
      "icon": "features-strongly-typed",
      "title": "Cross-platform",
      "content": "C# runs on almost any platform and chipset."
    },
    {
      "icon": "features-functional",
      "title": "Multi-paradigm",
      "content": "C# is primarily an object-oriented language, but also has lots of functional features."
    },
    {
      "icon": "features-lazy",
      "title": "General purpose",
      "content": "C# can be used for a wide variety of workloads, like websites, console applications, and even games."
    },
    {
      "icon": "features-declarative",
      "title": "Tooling",
      "content": "C# has excellent tooling, with linting and advanced refactoring options built-in."
    },
    {
      "icon": "features-generic",
      "title": "Documentation",
      "content": "Documentation is excellent and exhaustive, making it easy to get started with C#."
    }
  ]
}
```

## Tracking

https://github.com/exercism/v3-launch/issues/5

### PR DESCRIPTION
**Pull request by yamb00**
Add key features to track bash

resolve #469

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

**Comment by glennj** [comment]
Configlet failures: https://github.com/exercism/bash/actions/runs/5267025041/jobs/9537788883?pr=634

Do this and address the issues:
```sh
./bin/fetch-configlet
./bin/configlet lint
```

File: config.json [hunk 1]
```diff
@@ -1168,7 +1168,38 @@
...
+      "icon": "general-purpose"
+    },
+    {
+      "title": "Data Processing",
+      "content": "Bash provides powerful string manipulation capabilities and integration with other cli tools.",
+      "icon": "powerful"
+    },
+    {
+      "title": "Task Scheduling",
+      "content": "Bash scripts can be used with scheduling tools like cron to automate tasks at specific intervals.",
```
Comment (glennj) [inline config.json:1199]: Lots of other languages work with cron.

Some other key aspects of bash for your consideration:
* pipelines to glue utilities together
* bash is widely available: portable, cross-platform
* concurrency: background processes and bash coprocesses
* interactive: same language that you use interactively

**Review by glennj — CHANGES_REQUESTED** [review]

**Comment by glennj** [comment]
@yamb00 I hope I didn't scare you off. I'd love to see an update for your PR.

**Comment by yamb00** [comment]
> @yamb00 I hope I didn't scare you off. I'd love to see an update for your PR.



> @yamb00 I hope I didn't scare you off. I'd love to see an update for your PR.

I just had no time to work on this

**Review by glennj — APPROVED** [review]
Nice! Thanks for this.

**Comment by glennj** [comment]
@IsaacG want to weigh in?

**Pull request merged**